app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,26 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <%= yield %>
+
+    <%= render partial: "shared/navbar" %>
+
+    <div class="container">
+      <% if notice.present? %>
```

Comment:
The implementation for displaying notices and alerts using partials is effective for DRY (Don't Repeat Yourself) principles and enhances maintainability. Passing local variables for messages and CSS classes is a flexible way to reuse the partial for different message types.